components/dashboard/src/app/AppRoutes.tsx

```diff
@@ -265,8 +272,9 @@ export const AppRoutes: FunctionComponent<AppRoutesProps> = ({ user, teams }) =>
                                     switch (maybeProject) {
                                         case "projects":
                                             return <Projects />;
+                                        // TODO: Determine if we surface this - if so, can it also use WorkspacesNew?
```

Comment:
This is a left over from a past where we had /workspaces under teams. We are going to reintroduce this, soon, but for now it isn't really used. So removing it would be fine as well, but at least replacing is good, too.

Reply:
I'll go ahead and remove it then, thanks!